hs_core/urls.py

```diff
@@ -15,13 +15,13 @@
     url(r'^resourceList/$', views.resource_rest_api.ResourceList.as_view(),
         name='DEPRECATED_list_resources'),
 
-    url(r'^resource/$', views.resource_rest_api.ResourceListCreate.as_view(),
-        name='list_create_resource'),
-
```

Comment:
Why does this resource create REST endpoint be deleted? It appears to me the REST api class in resource_rest_api.py is still in our codebase and used for resource creation or listing.